.travis.yml

```diff
@@ -1,9 +1,22 @@
 # Copyright 2018 VMware, Inc.
 # SPDX-License-Identifier: BSD-2-Clause
 
-sudo: false
+dist: "trusty"
+
+install:
+  - yarn install
+  - pip3 install --user -r requirements.txt
+  - curl -sfL https://raw.githubusercontent.com/securego/gosec/master/install.sh | sh -s -- -b $GOPATH/bin 1.2.0
+  - export GOPATH=$GOPATH:/home/travis/build/vmware/precaution/test/fixtures/go
 language: node_js
 node_js:
-  - "8.3"
+  - "11"
```

Comment:
Can you put all 3 currently supported Node releases (8, 10 and 11)?
Travis will use the latest release for each so it should pass (previously it failed because ESlint requires 8.10 or higher).